html/config/install.md

```diff
@@ -0,0 +1,68 @@
+
+stable linux (debian) source julia
+apt-get install libboost-math-dev libboost-program-options-dev libboost-test-dev libboost-serialization-dev libarmadillo-dev binutils-dev libensmallen-dev libstb-dev wget !! wget -c http://mlpack.org/files/mlpack-latest.tar.gz -O - | tar -xz ! cd mlpack ! mkdir build ! cmake -DBUILD_JULIA_BINDINGS=ON .. ! make ! make install !! # In Julia ! using mlpack
+
+stable linux (debian) source all
+apt-get install libboost-math-dev libboost-program-options-dev libboost-test-dev libboost-serialization-dev libarmadillo-dev binutils-dev libensmallen-dev libstb-dev python-pandas python-numpy cython python-setuptools wget !! wget -c http://mlpack.org/files/mlpack-latest.tar.gz -O - | tar -xz ! cd mlpack ! mkdir build ! cmake -DBUILD_PYTHON_BINDINGS=ON -DBUILD_JULIA_BINDINGS=ON .. ! make ! make install 
+
+stable linux (debian) conda c++
+conda install -c conda-forge mlpack !! # Simple compiler command ! g++ code.cpp -larmadillo -lmlpack 
+
+stable linux (debian) pkg.jil julia
+# In Julia ! import Pkg; ! Pkg.add("mlpack") 
+
+stable macos brew c++
+brew install mlpack !! # Simple compiler command ! g++ code.cpp -larmadillo -lmlpack
+
+stable macos pkg.jil julia
+# In Julia ! import Pkg; ! Pkg.add("mlpack")
+ 
+stable macos source c++
+brew install boost armadillo ! wget -c http://mlpack.org/files/mlpack-latest.tar.gz -O - | tar -xz ! cd mlpack ! cd mlpack ! mkdir build ! cmake .. ! make ! make install !! # Simple compiler command ! clang++ code.cpp -larmadillo -lmlpack
+
+stable macos source julia
+brew install boost armadillo ! wget -c http://mlpack.org/files/mlpack-latest.tar.gz -O - | tar -xz ! cd mlpack ! cd mlpack ! mkdir build ! cmake -DBUILD_JULIA_BINDINGS=ON .. ! make ! make install !! # In Julia ! using mlpack
+
+stable macos source python
+brew install boost armadillo ! pip install numpy pandas cython ! wget -c http://mlpack.org/files/mlpack-latest.tar.gz -O - | tar -xz ! cd mlpack ! cd mlpack ! mkdir build ! cmake -DBUILD_PYTHON_BINDINGS=ON .. ! make ! make install !! # In Python ! import mlpack
+
+stable windows msi c++
+# See https://www.mlpack.org/doc/mlpack-3.2.2/doxygen/build_windows.html
+
+stable windows source c++
+# See https://www.mlpack.org/doc/mlpack-3.2.2/doxygen/build_windows.html
+
+nightly linux (debian) source c++
+apt-get install libboost-math-dev libboost-program-options-dev libboost-test-dev libboost-serialization-dev libarmadillo-dev binutils-dev libensmallen-dev libstb-dev wget git !! git clone https://github.com/mlpack/mlpack.git ! cd mlpack ! mkdir build ! cmake .. ! make ! make install !! # Simple compiler command ! g++ code.cpp -larmadillo -lmlpack
```

Comment:
This is a bit longer than the text box; maybe we should open an issue to automatically resize it if the text is too big?

Reply:
Yes, will add it to the issue.